packages/design-system/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -306,17 +306,20 @@ export const Autocomplete = (props: AutocompleteProps) => {
     menuProps['aria-labelledby'] = `${menuHeadingId} ${menuProps['aria-labelledby'] ?? ''}`;
   }
 
+  const menuContent = renderItems();
+
   return (
     <div className={rootClassName}>
       {renderChildren()}
 
       <div
-        className={classNames('ds-c-autocomplete__list', !isOpen && 'ds-u-display--none')}
+        className="ds-c-autocomplete__list"
         id={menuContainerId}
+        hidden={!(isOpen && menuContent)}
```

Comment:
Oh, this change came up during a pair programming session for the new Dropdown component. We can use a built-in browser attribute here rather than relying on our more verbose utility class